public/app/features/explore/state/time.ts

```diff
@@ -165,7 +165,6 @@ export const timeReducer = (state: ExploreItemState, action: AnyAction): Explore
       },
       isLive: live,
       isPaused: live ? false : state.isPaused,
-      loading: live,
```

Comment:
This was introduced in https://github.com/grafana/grafana/pull/18804 (reducers.ts file). Probably to clear loading state from logs panel when streaming is paused (in such case queryResponse.state is still `streaming`), but we don't have any loading state in logs panel while streaming, we may need to revisit it when switching to PanelChrome.